src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+thumbs.addEventListener('click', (even) => {
```

Comment:
There is a typo in the event parameter name. It should be 'event' instead of 'even'. This will help in maintaining consistency and clarity in the code.